src/error.rs

```diff
@@ -74,7 +74,7 @@ pub(super) enum Parse {
     Header(Header),
     TooLarge,
     Status,
-    #[cfg_attr(debug_assertions, allow(unused))]
+    #[allow(unused)]
```

Comment:
It's always unused? Then we should remove it. But I think it *is* used at times. I'd personally prefer we see the warning when that case is no longer true, then we can just delete the variant...

Reply:
Well , I may put more thought on that. Revert by now .